lib/controllers/patients/dump.js

```diff
@@ -3,10 +3,12 @@ var express         = require("express"),
     auth            = require("../helpers/auth.js"),
     generateDump    = require("../helpers/dump.js");
 
+const guard = auth.roleGuard;
+
 var dump = module.exports = express.Router({ mergeParams: true });
 
 // View JSON dump of all patient data the user has access to
-dump.get("/", auth.authenticate, auth.authorize("read"), function (req, res, next) {
+dump.get("/", auth.authenticate, guard(["admin", "programAdministator", "clinician", "user"]), auth.authorize("read"), function (req, res, next) {
```

Comment:
could this possibly reveal notes to the user if they requested a JSON dump? I don't know how this works

Reply:
Good catch! I should probably remove "user" from this.

Reply:
take programAdmin and user off for now